module/signer.go

```diff
@@ -24,5 +24,8 @@ type AggregatingSigner interface {
 type ThresholdSigner interface {
 	ThresholdVerifier
 	Sign(msg []byte) (crypto.Signature, error)
-	Combine(size uint, shares []crypto.Signature, indices []uint) (crypto.Signature, error)
+}
```

Comment:
What's the reason of removing the `Combine` method ? 

I guess the current `Combine` implementation does not use the receiver object, but the point of keeping it is to mirror the `AggregatingSigner` interface above. Both interfaces are evolving soon to stateful APIs where a receiver is needed for `Combine` and `Aggregate`.

Reply:
My thinking was that the Combine method doesn't require a private key, and it would be awkward to have to instantiate a threshold-signer (with a private key) just to perform this operation. Ex:
https://github.com/onflow/flow-go/blob/cdc8e61254d123071d4f0160f31a30136b7a665d/consensus/hotstuff/verification/combined_signer.go#L164

Reply:
Yes that makes sense with the current stateless implementation of threshold signing. 
We're moving to a stateful one soon and the same object will be the one signing, combining and verifying. 
We can keep your current branch this way, but one of us will have to make a refactor before merging to master later, whoever is slower 😅

Reply:
sounds good:)